File: python/max.py

print("To find max of 10 number's:")
Numbers = []
Counter = 0
print("Enter the first number:", end = '')
Numbers.append(int(input()))
while Counter < 9:
    print("Enter the next number:", end = '')
    Numbers.append(int(input()))
    Counter = Counter + 1
print("The max of", end = '')
Counter = 0;
while Counter < 10:
    print("", Numbers[Counter], end = '')
    Counter = Counter + 1
print(" is", end = '')
Max = Numbers[0]
for Counter in range(0,10):
    if Max < Numbers[Counter]:
       Max = Numbers[Counter]
print('', Max, end = '.')
 
  

File: Swift/max.swift
import Foundation
print("To find max of 10 number's:")
var Counter = 0
var Numbers = [Int]()
print("Enter the first number", terminator:":")
Numbers.append(Int(readLine()!)!)
for _ in 1...9
{
    print("Enter the next number", terminator:":")
    Numbers.append(Int(readLine()!)!)
}
print("The max of ", terminator : "")
var Count = 0 
for Count in 0...9
{
  print("\(Numbers[Count])", terminator : " ")
}
var C = 0
var Max = Numbers[0]
for C in 1...9
{
  if(Max < Numbers[C])
  {
    Max = Numbers[C]
  }
}
print("is \(Max).")